src/main/java/org/veupathdb/service/eda/compute/plugins/differentialabundance/DifferentialAbundancePlugin.java

```diff
@@ -55,7 +55,8 @@ protected void execute() {
     EntityDef entity = meta.getEntity(entityId).orElseThrow();
     VariableDef computeEntityIdVarSpec = util.getEntityIdVarSpec(entityId);
     String computeEntityIdColName = util.toColNameOrEmpty(computeEntityIdVarSpec);
-    String method = computeConfig.getDifferentialAbundanceMethod().getValue();
+    // if we ever introduce ANCOM-BC or something else we'll need to change this
```

Comment:
Should we instead just change the argument options in microbiomeComputations?

Reply:
oh no i read this opposite. We don't want to change the api. I see.
Should we? Or expand it so that either sending "Maaslin" or "Maaslin2" gives the same result?

Reply:
im not sure im following this comment. i want the mbio computes api to always include version numbers, bc people might be looking at them and care in that context and so i dont want them to be vague. in this context i dont think anyone will care, and actively dont want to break cached things, so the compute service api i want to stay the same.